services/sourceforge/sourceforge-tickets.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  count: nonNegativeInteger.required(),
+}).required()
+
+module.exports = class SourceforgeTickets extends BaseJsonService {
+  static get category() {
+    return 'other'
+  }
+
+  static get route() {
+    return {
+      base: 'sourceforge/tickets',
```

Comment:
Also, should this perhaps use `open-tickets` for the noun? I know the example listing include "open", but I think it's probably worth making that explicit in the badge route:

```suggestion
      base: 'sourceforge/open-tickets',
```

Reply:
If you don't mind, I'd prefer to integrate this in a future PR.

Reply:
Sure that sounds good. In that case though, I do still think that the noun here should be `open-tickets` or `tickets-open` to make that more explicit. If there's any follow up work to support open/closed/total then we can add redirectors if needed

Reply:
@Prouser123 could you please make this small change? I believe it's the only blocker for merging this. 😄

Reply:
Really sorry about this. I made this change locally when offline and never pushed it to the remote. Oops!